web/src/message_notifications.js

```diff
@@ -98,8 +98,9 @@ function get_notification_title(message, content, msg_count) {
         case "private":
             other_recipients = remove_sender_from_list_of_recipients(message);
             if (message.display_recipient.length > 2) {
+                // Character limit taken from https://www.pushengage.com/push-notification-character-limits
                 // If the message has too many recipients to list them all...
-                if (content.length + title.length + other_recipients.length > 230) {
+                if (title.length + other_recipients.length > 30) {
```

Comment:
I think this change will feel bad on the non-mobile platforms where the limit is substantially above 30 characters.

Reply:
Note that this only happens if there are more than 2 recipients.

So, new notifications would looks like this most of the time:
<img width="388" alt="Screenshot 2023-10-23 at 7 17 32 AM" src="https://github.com/zulip/zulip/assets/25124304/b6697e5b-1c4b-43e3-9124-a6e5c637b778">
<img width="388" alt="Screenshot 2023-10-23 at 7 17 43 AM" src="https://github.com/zulip/zulip/assets/25124304/7758f4af-9e8e-4178-b367-086c012c2bb6">

Which is more informative than
<img width="388" alt="Screenshot 2023-10-23 at 7 19 25 AM" src="https://github.com/zulip/zulip/assets/25124304/a57c3ee0-23cf-4cd4-8eca-6732f58bd839">

So, it having a low limit set here actually helps us get a better notification for the desktop devices.

Reply:
or maybe not, thinking more about it, maybe having even a portion of 3rd can help you distinguish which group DM is the conversation a part of.

Reply:
What if we make "and 2 other people" shorter, to try to squeeze another name in if we can? Something like this should work:

> Iago (to you, Polonius, +2 more)

Reply:
I think that's unlikely to work in general; most people's full names take a substantial portion of 30 characters, such that 2 names will often exceed that. But possibly would work in languages like Chinese where names are a very small number of characters.

Reply:
I updated this to use 50 as limit. Happy to make any other changes to unblock https://github.com/zulip/zulip/issues/21032.